source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -37,3 +37,28 @@ payment flow. Otherwise, it will expire.
 If your service has enabled payment receipt emails, GOV.UK Pay will send a
 payment receipt email to the user once your service's capture request is
 received and accepted.
+
+## See the capture URL for a payment
+
+If a payment is available for capture, you can see its capture URL in 
+GOV.UK Pay API responses such as:
+
+* the response to a create payment request
```

Comment:
A payment will never be available for capture when it’s first created (it will have `"capture": null`), so I don’t think it’s worth having this item in the list